axon_config.go

```diff
@@ -25,7 +37,31 @@ type AxonConfig struct {
 	// between source and target.
 	StartFromOffset int64 `envconfig:"start_from_offset" default:"0"`
 
+	// Start replication from the specified logical sequence number. (LR mode only)
```

Comment:
What linter are we using? I am surprised it is not complaining about docs not starting with `StartFromLSN`.

Reply:
I didn't see any linters built in to the repo, so I whipped up #47 
Though it doesn't look like that is currently catching exported docstring errors...

Reply:
Ah yeah. `golangci-lint`... I was beating my head against the desk trying to get this to work yesterday. Needs `revive` linter enabled, and some excludes disabled afaik, but it still did not work! You can enable `golint` but it complains about it being deprecated 🤦